source/features/index.tsx

```diff
@@ -1,9 +1,11 @@
 import React from 'dom-chef';
 import select from 'select-dom';
 import domLoaded from 'dom-loaded';
+import cache from 'webext-storage-cache';
 import stripIndent from 'strip-indent';
 import {Promisable} from 'type-fest';
 import elementReady from 'element-ready';
+import looseVersionCompare from 'tiny-version-compare';
```

Comment:
```suggestion
import compareVersions from 'tiny-version-compare';
```

This matches https://github.com/sindresorhus/refined-github/blob/0a58069c118954556d667c661d8cee31ff826400/source/github-helpers/index.ts#L3

Reply:
there is also an `import tinyVersionCompare from 'tiny-version-compare';` in `source/features/tag-changelog-link.tsx` but I agree `compareVersions` is better than `{tiny,loose}VersionCompare`